src/app/containers/FastBtcDialog/components/TopUpHistory.tsx

```diff
@@ -26,7 +26,7 @@ export function TopUpHistory() {
   const account = useAccount();
   const state = useSelector(selectFastBtcDialog);
   const dispatch = useDispatch();
-
+  const [history, setHistory] = useState([]) as any;
```

Comment:
1. type is in wrong place, should be useState`<any>`([])
2. we don't use `any` anymore - prepare dedicated typing for history item